NuGetPackageVerifier.json

```diff
@@ -17,54 +17,53 @@
       "DefaultCompositeRule"
     ],
     "packages": {
-      "Microsoft.AspNetCore.Razor.Design": {
+      "Microsoft.NET.Sdk.Razor": {
         "exclusions": {
           "ASSEMBLY_DESCRIPTION": {
-            "tools/Microsoft.CodeAnalysis.CSharp.dll": "This assembly is not owned by us and does not follow our conventions.",
-            "tools/Microsoft.CodeAnalysis.dll": "This assembly is not owned by us and does not follow our conventions."
+            "tools/netcoreapp2.0/Microsoft.CodeAnalysis.CSharp.dll": "This assembly is not owned by us and does not follow our conventions.",
+            "tools/netcoreapp2.0/Microsoft.CodeAnalysis.dll": "This assembly is not owned by us and does not follow our conventions."
```

Comment:
Do we need to ship these here now that we are in the SDK? I'm not sure how all of this stuff gets constructed.

Reply:
Filed https://github.com/aspnet/Razor/issues/2675 to figure this out.